connectors/storage/s3.py

```
@@ -0,0 +1,189 @@
+    def read_file(self, file_path, **kwargs):
+        ext = os.path.splitext(file_path)[1].lower()
+        obj = self.client.get_object(Bucket=self.bucket, Key=file_path)
+        content = obj["Body"].read()
+
+        if ext == ".csv":
+            return pd.read_csv(io.BytesIO(content))
+        elif ext == ".json":
+            return pd.read_json(io.BytesIO(content))
+        elif ext == ".parquet":
+            import pyarrow.parquet as pq
+            import pyarrow as pa
```

Ruff (0.12.2)

`pyarrow` imported but unused

Remove unused import: `pyarrow`

(F401)